### ISSUE TITLE
inAppbrowser.executeScript not working in Android

### ISSUE THREAD

**Issue by gatelli** [apache/cordova-plugin-inappbrowser#402]
Hi,
I need to use executeScript like :
`inAppRef.addEventListener('loadstop', function() {
inApp.executeScript({
                    code: "document.getElementsByTagName('html')[0].innerHTML"
                  }, function(html) {
// my code using html
}
}`
This is working well with iOS but not with Android. The loadstop is triggered but not the function(html).

**Comment by danova123** [comment]
Facing the same issue. It was working earlier though
```
 inAppBrowserRef = cordova.InAppBrowser.open(url, target, options);
 inAppBrowserRef.addEventListener('loadstart', loadStartCallBack);
 function loadStartCallBack() {
            inAppBrowserRef.executeScript({ code: code },executeScriptCallBack);
          }
 function executeScriptCallBack(params){
    alert("ExecuteScriptCallBack called");   ///THIS NEVER GETS CALLED

}
```

**Comment by danova123** [comment]
Same issue facing here as well. It used to work earlier.

**Comment by ianhowe76** [comment]
Is the `html` data in the callback large?

There is an issue on Android since Chrome v 69.0.3497.91 that truncates the data to 10240 bytes. If your callback data is greater than 10240 bytes the plugin will have an error and the callback will not be called

See #303 for details and fix (awaiting merge)

**Comment by gatelli** [comment]
For my case, the callback data is >10240 bytes. I had already tried this PR by merging the 2 changed files but it's not working for me unfortunately.

**Comment by danova123** [comment]
@ianhowe76 you are correct, if i just fetch one element by getElementById it works. Any idea, when your changes will be merged

**Comment by ianhowe76** [comment]
@danova123 Not sure how fast PRs get merged for this project. Maybe put a comment in https://github.com/apache/cordova-plugin-inappbrowser/pull/395 saying that you tested it ok

**Comment by danova123** [comment]
@ianhowe76 Unfortunately its not working for me too. 
Steps:

`cordova plugin add https://github.com/ianhowe76/cordova-plugin-inappbrowser.git`

then
`inAppRef.addEventListener('loadstop', function() { inApp.executeScript({ code: "document.getElementsByTagName('html')[0].innerHTML" }, function(html) { // my code using html } }
`

**Comment by aigenseer** [comment]
@gatelli 
maybe you have chanced the file "plugins/cordova-plugin-inappbrowser/src/android/InAppBrowser.java",
but you have to chance the "platforms/android/app/src/main/java/org/apache/cordova/inappbrowser/InAppBrowser.java" file.
Or you download the source code, chance the lines and add the new source code in your cordova project.

**Comment by casper123** [comment]
@ianhowe76  it doesn't work for me, I'm on Android 9 and Chrome version 74

**Comment by gatelli** [comment]
> @gatelli
> maybe you have chanced the file "plugins/cordova-plugin-inappbrowser/src/android/InAppBrowser.java",
> but you have to chance the "platforms/android/app/src/main/java/org/apache/cordova/inappbrowser/InAppBrowser.java" file.
> Or you download the source code, chance the lines and add the new source code in your cordova project.

Yes it's working now. Many thanks !

**Comment by Hanzofm** [comment]
It not works for me, is there any solution?
